lib/checks/isQuadratic.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
```

Comment:
can you merge in the new linter changes? apparently we don't need `'use strict'` in modules because they're strict by default (the linter will catch this now 😄  )